descriptors/ModuleDescriptor-template.json

```diff
@@ -1057,12 +1057,21 @@
     },
     {
       "id": "oaipmhview",
-      "version": "1.0",
+      "version": "1.1",
       "handlers" : [
         {
           "methods" : [ "GET" ],
           "pathPattern" : "/oai-pmh-view/instances",
           "permissionsRequired": ["inventory-storage.oai-pmh-view.instances.collection.get"]
+        },
+        {
+          "methods" : [ "GET" ],
+          "pathPattern" : "/oai-pmh-view/updatedInstanceIds",
+          "permissionsRequired": ["inventory-storage.oai-pmh-view.updatedinstanceids.collection.get"]
+        }, {
+          "methods" : [ "POST" ],
+          "pathPattern" : "/oai-pmh-view/enrichedInstances",
```

Comment:
What are enriched instances?

Reply:
It just contains instance data (items with holdings as in former and currently retained API entry, for backward compatibility purposes) but without some atttributes on the top level (like "lastUpdatedDate" and "deleted"). Indeed possibly it can be named differently, but I belive to so as not not confuse with former get@"/oai-pmh-view/instances" possibly it would be named differently. Or maye taking into account another method the name can be retaned - post@"/oai-pmh-view/instances". How do you think, which approach will be better?

Reply:
If the name is intended to keep consistency with existing decisions, I suggest it be left as is.